src/V3Coverage.cpp

```diff
@@ -39,6 +42,15 @@ class CoverageVisitor final : public VNVisitor {
     // TYPES
     using LinenoSet = std::set<int>;
 
+    struct CoverExpr final {
+        AstNodeExpr* m_expr;
```

Comment:
```suggestion
        AstNodeExpr* const m_exprp;
```

And comment members (as to what the expression represents)
Think can be const, below also.